rules-testing/test/recipes-test.js

```diff
@@ -0,0 +1,236 @@
+const assert = require('assert');
+const firebase = require("@firebase/testing");
+// var expect = require('chai').expect;
+
+const MY_PROJECT_ID = "boq-devex-step-2020";
```

Comment:
Normally it'd be good to put special values like this in some common config place so that it's easy to rename everywhere if/when needed, but this seems fine given the circumstances